syntax = "proto3";
package monitor.proto;

message UserMessage {
    string account_num = 1;
    string pwd = 2;
}

message UserResponseMessage {
    string response_str = 1;
    repeated string machine_name_array = 2;
}
